scripts/Override_respot.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
What is this scripts required for?

Reply:
Started implementing the res potential linker to allow for custom Res potential data